### PR TITLE
Fix "RET503", # Missing explicit `return` at the end of function able to return non-`None` value

### PR DESCRIPTION
**Pull request by jiridanek**

### CODE REVIEW

**Review by jstourac — APPROVED** [review]

**Review by kobihk — DISMISSED** [review]

File: ods_ci/utils/scripts/ocm/ocm.py [hunk 1]
```diff
@@ -642,6 +642,7 @@ def install_rhoam_addon(self, exit_on_failure=True):
             #    self.wait_for_addon_installation_to_complete(addon_name="managed-api-service")
         else:
             log.info("managed-api-service is already installed on {}".format(self.cluster_name))
+            return None
```
[the remark this answers is not in the document]
Comment (jiridanek) [reply ods_ci/utils/scripts/ocm/ocm.py]: returning True, thanks

**Review by kobihk — APPROVED** [review]

**Comment by dhirajsb** [comment]
I'm not sure why I'm being asked to review this? Did you mean to request a review from someone else?

**Comment by jiridanek** [comment]
> I'm not sure why I'm being asked to review this? Did you mean to request a review from someone else?

I clicked all the autosuggestions from github, should be based on who edited the files recently. It's possible i missclicked. If so, sorry about the disturbance.

**Comment by jstourac** [comment]
@jiridanek has this been tested indeed? I can see following failure in our job :thinking: 

```
07:30:59  + python3 /home/jenkins/workspace/rhods/exploratory-test-rhods-deploy/ods-ci/ods_ci/utils/scripts/ocm/ocm.py ocm_login --token ****
07:30:59  Traceback (most recent call last):
07:30:59    File "/home/jenkins/workspace/rhods/exploratory-test-rhods-deploy/ods-ci/ods_ci/utils/scripts/ocm/ocm.py", line 28, in <module>
07:30:59      class OpenshiftClusterManager:
07:30:59    File "/home/jenkins/workspace/rhods/exploratory-test-rhods-deploy/ods-ci/ods_ci/utils/scripts/ocm/ocm.py", line 1186, in OpenshiftClusterManager
07:30:59      def change_cluster_channel_group(self) -> str | None:
07:30:59  TypeError: unsupported operand type(s) for |: 'type' and 'NoneType'
```

**Comment by jiridanek** [comment]
The initial version was semantically equivalent code. The changes I did at @kobihk 's suggestion do have potential to break things. And no, I did not test it before merge.

**Comment by kobihk** [comment]
> The initial version was semantically equivalent code. The changes I did at @kobihk 's suggestion do have potential to break things. And no, I did not test it before merge.

@jiridanek maybe without the definition of return type it could work

**Pull request merged**